File: src/android/com/kuackmedia/androidauto/models/AlbumSummary.kt
package com.kuackmedia.androidauto.models

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class AlbumSummary(
  val id: Long,
  val upc: String? = null,
  val score: Double? = null,
  val title: String? = null,
  val active: Boolean? = null,
  val images: List<CoverImage>? = null,
  val artists: List<Artist>? = null,
  val itemType: String? = null,
  val subTitle: String? = null,
  val tracksQty: Int? = null,
  val releaseDate: String? = null,
  val imageColorInfo: ImageColorInfo? = null
)

File: src/ios/models/AlbumSummary.swift
import Foundation

struct AlbumSummary: Codable {
    let id: Int64
    let upc: String?
    let score: Double?
    let title: String?
    let active: Bool?
    let images: [CoverImage]?
    let artists: [Artist]?
    let itemType: String?
    let subTitle: String?
    let tracksQty: Int?
    let releaseDate: String?
    let imageColorInfo: ImageColorInfo?
}
